docs/headless-cms/docs/01-01-headless-cms.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Overview
+---
+
+The Headless CMS is a new breed of traditional Content Management Systems (CMS) that provides a way of storing and managing raw content, and exposing it through an API. It allows you to pull this content into your own application and tailor it to your needs, delivering it in any format, on any device. Contrary to the traditional CMS, such as WordPress, the Headless CMS does not provide a display layer and ready-to-use templates. Instead, it only ensures the backend in the form of a database and gives flexibility on the choice of the frontend, thus cutting the default "head" off the traditional CMS solutions. 
+
+## Headless CMS in Kyma
+
+Kyma takes the concept of the Headless CMS into a higher, cloud-native level. It provides a Kubernetes-based solution that relies on the custom resource (CR) extensibility feature. You add a document to the Console UI through a CR, while the backend mechanism in the form of the [Asset Store](#asset-store-overview) handles the rest. While the Asset Store itself allows you to add separate Asset CRs and store them in respective Minio buckets, the Headless CMS ensures the upload of multiple and grouped data for a given documentation topic. All you need to do is to specify all details of the topic, including the documentation sources, in a DocsTopic CR or a ClusterDocsTopic and apply it to a given Namespace or a cluster, respectively. The CR supports various documentation formats, including images, Markdown documents, AsyncAPI, OData, and OpenAPI specification files. You can upload them both as single and packed (ZIP or TAR) assets. The labels defined in the CR specify in which view and under which group the Console UI renders the assets.
```

Comment:
Which sentence exactly?

Reply:
`Kyma takes the concept of the Headless CMS into a higher, cloud-native level`
it sounds nice but doesn't explain anything, better for release notes then documentation. Docs should only have content that brings value to the reader. I think that if you remove it, the whole paragraph will not loos anything because of it. So what I mean is, less text is better as reader is more encouraged to read and take something from it

Reply:
Actually, I did it a bit on purpose but if you consider it redundant, will remove it.